src/App.tsx

```diff
@@ -1,12 +1,93 @@
 import React from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+interface State {
+  goods: Good[],
+}
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
 
-export default App;
+  getGoodsHandler = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
you can get on props needed callback (getAll, get5First, or getRedGoods) and call it